addons/mail/static/src/core/common/thread.xml

```diff
@@ -92,9 +90,7 @@
 </t>
 
 <t t-name="mail.NotificationMessage">
-    <div class="o-mail-NotificationMessage text-break mx-auto text-500 small px-3 text-center" t-on-click="onClickNotification" t-att-class="{
-        'mt-1': prevMsg and !prevMsg.isNotification,
-    }" t-ref="root">
+    <div class="o-mail-NotificationMessage text-break mx-auto text-500 small px-3 text-center" t-on-click="onClickNotification" t-att-class="props.className" t-ref="root">
```

Comment:
I would `<-- @this {common/message} -->` above as to understand context of template is in message component, especially if we keep template here for stable reasons

Reply:
Indeed, it would be easier to understand.